### PR TITLE
Aem Author Sync Alarm

### PR DESCRIPTION
**Pull request by nletts**

### CODE REVIEW

File: ansible/playbooks/apps/author.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@
           InboundFromBastionHostSecurityGroupParameter: "{{ compute.inbound_from_bastion_host_security_group }}"
           DataBucketNameParameter: "{{ s3.data_bucket_name }}"
           AemAwsStackProvisionerVersionParameter: "{{ library.aem_aws_stack_provisioner_version }}"
+          AemAuthorSyncDelayAlarmTopicName: "{{ stack_prefix }}-{{ author.aem_sync_alarm.topic_name }}"
```
Comment (michaeldiender-shinesolutions) [inline ansible/playbooks/apps/author.yaml]: suffix with 'Parameter' for consistency with other parameters

File: cloudformation/apps/author.yaml [hunk 2]
```diff
@@ -85,6 +85,14 @@ Parameters:
     Type: String
     Description: AEM AWS Stack Provisioner version number
 
+  AemAuthorSyncDelayAlarmTopicName:
```
Comment (michaeldiender-shinesolutions) [inline cloudformation/apps/author.yaml]: suffix with 'Parameter' for consistency with other parameters

**Comment by nletts** [comment]
The parameter suffix has been added as requested

File: cloudformation/apps/author.yaml [hunk 3]
```diff
@@ -93,6 +101,31 @@ Conditions:
 
 Resources:
 
+  AemAuthorSyncDelayAlarmTopic:
```
Comment (michaeldiender-shinesolutions) [inline cloudformation/apps/author.yaml]: can we move the topic to the messaging stack.

File: cloudformation/apps/author.yaml [hunk 4]
```diff
@@ -93,6 +101,31 @@ Conditions:
...
 
+  AemAuthorSyncDelayAlarmTopic: 
+    Type: AWS::SNS::Topic
+    Properties: 
+      Subscription: 
+        - Endpoint: !Sub ${AemAuthorSyncDelayAlarmContactEmailParameter}
+          Protocol: email
+      TopicName: !Sub ${AemAuthorSyncDelayAlarmTopicNameParameter}
+
+  AemAuthorSyncDelayAlarm:
```
Comment (michaeldiender-shinesolutions) [inline cloudformation/apps/author.yaml]: for naming consistency with other resources. remove the Aem from 'AemAuthorSyncDelayAlarm'  the Author part assumes that it is aem.

**Comment by nletts** [comment]
Ok, I think I fixed everything. Please re-review

**Comment by michaeldiender-shinesolutions** [comment]
Code looks ok. Have you been able to create a new stack with the changes?

**Comment by nletts** [comment]
Yes, I've created a new stack with no issues

**Pull request merged**